### PR TITLE
Scala steward configuration

### PR DESCRIPTION
**Pull request by tpetillot**
chore: use circe version for circe-fs2
---

**Motivation:**
Must be aligned

chore: add scala steward configuration

---

**Modification:**
* freeze `cats-effect` to major version 2
* freeze `fs2-core` to major version (next use CE3)
* freeze `circe-fs2` to minor version (next use fs2 > v3)

### CODE REVIEW

File: .scala-steward.conf [hunk 1]
```diff
@@ -0,0 +1,5 @@
+updates.pin = [
+  { groupId = "co.fs2", artifactId="fs2-core", version="x." },
+  { groupId = "io.circe", artifactId="circe-fs2", version="x.y" },
+  { groupId = "org.typelevel", artifactId="cats-effect", version="x." },
```
Comment (joroKr21) [inline .scala-steward.conf:4]:
```suggestion
  { groupId = "co.fs2", artifactId="fs2-core", version="2." },
  { groupId = "io.circe", version="0.13." },
  { groupId = "org.typelevel", artifactId="cats-effect", version="2." },
```

**Review by joroKr21 — CHANGES_REQUESTED** [review]
Another thing I wonder about is if we should support both Cats Effect 2.x and 3.x on separate branches or just aim to move to 3.x soonish.

**Comment by tpetillot** [comment]
3.x soonish sound good to me (it'll motivate us to do the upgrade, and quit quill)

**Comment by tpetillot** [comment]
I think we're good for a release after this one? (which is not required into the release at all)

File: .scala-steward.conf [hunk 2]
```diff
@@ -0,0 +1,5 @@
+updates.pin = [
+  { groupId = "co.fs2", artifactId="fs2-core", version="2." },
+  { groupId = "io.circe", artifactId="circe-fs2", version="0.13." },
```
Comment (joroKr21) [inline .scala-steward.conf:3]: I think we need to pin all of Circe
```suggestion
  { groupId = "io.circe", version="0.13." },
```

**Comment by joroKr21** [comment]
> I think we're good for a release after this one? (which is not required into the release at all)

Yeah, feel free to cut a release 👍 

> Then, I may check what's needed to upgrade to CE3

Awesome!

**Review by joroKr21 — APPROVED** [review]

**Comment by tpetillot** [comment]
756cea1a61ae82418ba7c3413f05448d71dde402 release doesn't seems to be triggered...

**Pull request merged**